eng/Version.Details.xml

```diff
@@ -30,53 +30,53 @@
       <Uri>https://github.com/dotnet/templating</Uri>
       <Sha>7a5e55ed9d9ea9061cbc60b2497e594fe12c480d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.0-preview.1.21071.15">
```

Comment:
We'll want to revert all the changes in the `eng` folder unless @pranavkm's changes need dependency updates